core_backend/app/contents/models.py

```diff
@@ -116,6 +113,53 @@ def __repr__(self) -> str:
         )
 
 
+class ContentForQueryDB(Base):
+    """
+    ORM for storing what content was returned for a given query.
+    Allows us to track how many times a given content was returned in a time period.
+    """
+
+    __tablename__ = "content_for_query"
```

Comment:
btw, we decided to use dashes for table names instead of underscores

Reply:
Leaving this as in as discussed in the tech session